### PR TITLE
Add dbconnect

### PR DESCRIPTION
**Pull request by akeym**

### CODE REVIEW

File: install_dbx.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+   - name: install packages for dbconnect addon
+     apt: name={{item}} state=present update_cache=yes
+     with_items:
+       - openjdk-8-jre
+
+   - name: set up java home envvar
+     lineinfile:
+       - path: /etc/environment
+       - line: 'JAVA_HOME="/usr/lib/jvm/java-8-openjdk-amd64"'
+       - state: present
```
Comment (stacybrock) [inline install_dbx.yml:15]: Is this the first and only addon that needs the JRE? Will this clobber any existing `JAVA_HOME`s?

**Review by stacybrock — COMMENTED** [review]
I'm assuming the contents of the `*.conf` files are correct. (I eyeballed them for sanity and obvious typos, but I don't know enough about SCCM or this addon to check each config entry for correctness.)

Hardcoded config in a public repo? Woe unto those who `git clone` this wantonly...

File: install_dbx.yml (hunk 1 above)
Comment (akeym) [reply install_dbx.yml:15]: The first we've run across, yes.  I'm not sure what the future holds though..

**Review by stacybrock — APPROVED** [review]
LGTM

**Pull request merged**